source/_integrations/imap.markdown

```diff
@@ -226,7 +226,7 @@ template:
         id: "custom_event"
         event_data:
           sender: "no-reply@smartconnect.apc.com"
-  - sensor:
+    sensor:
```

Comment:
The sensors below/in this list need to be indented in that case (to match our YAML style guide).

Reply:
Nice catch